cypress/integration/User Management/BJ_00_InstructorEdit.js

```diff
@@ -0,0 +1,130 @@
+                        "data": {
+                            "createInstructor": {
+                                "created": false,
+                            },
+                        },
+                    },
+                    "test": (variables) => {
+                        Object.entries(variables).forEach(([key, value]) => {
+                            if (!["phoneNumber", "id", "user", "password", "subjects", "language"].includes(key)) {
+                                expect(instructor2[key] || instructor2?.user[key]).equals(value);
+                            }
+                        });
+                    },
+                },
+                "MyMutation": {
+                    "response": {
+                        "data": {
+                            "inviteInstructor": {
+                                inviteInstructorConfirmation
+                            }
+                        }
+                    },
+                },
+                "GetInstructor": {
+                    "response": {
+                        "data": {
+                            instructor
+                        },
+                    },
+                    "test": ({userID}) => {
+                        expect(userID).equals(instructor.user.id.toString(), "Check ID passed");
+                    },
+                },
+            });
+            cy.visitAuthenticated(`/form/instructor/${instructor.user.id}`);
+        });
+    });
+
+    it("Loads data properly into the first form", () => {
+        cy.fixture("users.json").then(({instructor}) => {
+            cy.get("[data-cy=basicInfo-firstName-input]")
+                .should("have.value", instructor.user.firstName);
+            cy.get("[data-cy=basicInfo-address-input]")
+                .should("have.value", instructor.address);
+            cy.get("[data-cy=basicInfo-phoneNumber-input]")
+                .should("have.value", instructor.phoneNumber);
+            cy.get("[data-cy=basicInfo-nextButton]").should("be.enabled");
+        });
+    });
+
+    it("Can enter data properly into the first form", () => {
+        cy.fixture("users.json").then(({instructor2}) => {
+            cy.get("[data-cy=basicInfo-phoneNumber-input]")
+                .clear()
+                .fastType("0");
+            cy.get("[data-cy=basicInfo-nextButton]").should("be.disabled");
+            cy.get("[data-cy=basicInfo-phoneNumber-input]").clear();
+            cy.get("[data-cy=basicInfo-phoneNumber-input]").fastType(instructor2.phoneNumber);
+            cy.get("[data-cy=basicInfo-address-input]")
+                .clear()
+                .fastType(instructor2.address)
+                .should("have.value", instructor2.address)
+            cy.get("[data-cy=basicInfo-nextButton]").should("be.enabled");
+        }); 
+    });
+
+    it("Properly goes to the next part of the form and checks for data to load properly on the second form", () => {
```

Comment:
Be specific - what is "the next part of the form" ? use the specific section name. what "data" should be loading? what is the "second form"?